main.go

```diff
@@ -8,19 +8,32 @@ import (
 	"log"
 	"os"
 	"strings"
+	"strconv"
+	"reflect"
 
 	"github.com/bluele/slack"
 	"github.com/sensu/sensu-go/types"
 	"github.com/spf13/cobra"
 )
 
+type HandlerConfig struct {
```

Comment:
Makes sense. Are there other benefits of this approach other than styling?

Reply:
Good question! Yes. By avoiding reflection, the code relies only on information known at compile-time, instead of at run-time. This decreases the defect rate, as there is less going on at runtime.

While struct tags are present at compile time, the way to detect them involves reflection, which is rather tricky to get right.